Cargo.toml

```diff
@@ -98,8 +116,7 @@ name = "vector3"
 harness = false
 
 [profile.bench]
-debug = true
-# codegen-units = 1
+codegen-units = 1
```

Comment:
I am a bit reluctant to enable this by default. Generally I prefer to just use the defaults because I don't want to assume that projects are using any particular compiler settings.

Reply:
ah yeah.. i mostly did this cause nalgebra was being slower than i thought it should on my laptop and it fixed that.. it doesn't actually have an affect on my desktop though 🤷 

in any case I'm fine with reverting this